ps_facetedsearch.php

```diff
@@ -165,6 +165,10 @@ protected function getDefaultFilters()
                 'label' => 'Product price filter (slider)',
                 'slider' => true,
             ],
+            'layered_selection_review_star' => [
```

Comment:
This option should be available only when productcomments is installed.

Reply:
yes, will add.

Reply:
done